src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,15 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
```

Comment:
it's better to use `span`
```suggestion
      <span> by </span>
```